service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -368,4 +371,33 @@ private String getFormattedRemovalAddress(CaseData caseData) {
         return Optional.ofNullable(caseData.getEpoRemovalAddress())
             .map(address -> address.getAddressAsString(", ")).orElse("");
     }
+
+    public boolean shouldGenerateDocument(OrderTypeAndDocument orderTypeAndDocument,
+                                          FurtherDirections orderFurtherDirections,
+                                          List<Element<Child>> children,
+                                          String closeCaseFromOrder) {
+        // generate order if one of the following is met:
```

Comment:
Think I would prefer /** * * */ annotation for blocks of comments, and maybe above the method name rather than inside the method (this seems to be the approach we have in the code i.e CommonDirectionService

Reply:
Sure thing